test/functional/taxonomy_navigation_test.rb

```diff
@@ -0,0 +1,89 @@
+require "test_helper"
+
+class FakeController < ApplicationController
```

Comment:
Would it make sense to call this FakeTaxonomyNavigationController? Someone else might need a fake controller in future for similar purposes.